relayer-cli/src/commands/config/auto.rs

```diff
@@ -0,0 +1,200 @@
+use crate::chain_registry::get_configs;
+use abscissa_core::{
+    clap::Parser,
+    {Command, Runnable},
+};
+
+use crate::conclude::Output;
+
+use ibc_relayer::{
+    config::{store, ChainConfig, Config},
+    keyring::{KeyRing, Store::Test},
+};
+
+use std::path::PathBuf;
+use tokio::runtime::Builder;
+use tracing::{info, warn};
+
+fn find_key(chainconfig: &ChainConfig) -> Option<String> {
+    if let Ok(keyring) = KeyRing::new(Test, &chainconfig.account_prefix, &chainconfig.id) {
+        if let Ok(keys) = keyring.keys() {
+            if let Some((name, _)) = keys.first() {
+                Some(name.to_string())
+            } else {
+                None
+            }
+        } else {
+            None
+        }
+    } else {
+        None
+    }
+}
+
+/// The data structure that represents the arguments when invoking the `config auto` CLI command.
+///
+/// The command has two required arguments and an optional one which is used to manually specify the keys to use in the config file:
+///
+/// `config auto [OPTIONS] --path <PATH> --chain <CHAIN_NAME_1 CHAIN_NAME_2...>`
+///
+/// OR
+///
+/// `config auto [OPTIONS] --path <PATH> --chain <CHAIN_NAME_1 CHAIN_NAME_2...> --keys <KEY_NAME_1 KEY_NAME_2...>`
```

Comment:
I see how they are useful here for providing the corresponding keys in the same order, but perhaps we could improve the ux a little bit in this case by asking for `CHAIN:KEY` pairs? eg.

```
hermes config auto [OPTIONS] --path <PATH> --keys <CHAIN_1:KEY_1 CHAIN_2:KEY_2...>
```

What do you think?

Reply:
Nice idea! However, maybe we should only keep `--chains`. The command would be :
```
hermes config auto [OPTIONS] --path <PATH> --chains <CHAIN_1[:KEY_1] CHAIN_2[:KEY_2]...>
```
If a key is provided then it will be used, otherwise the command would check if a key already exists otherwise it would leave a blank `key_name` argument.